voting-webapp/application/results-exporter.yaml

```diff
@@ -29,7 +29,7 @@ spec:
     spec:
       containers:
       - name: result-exporter
-        image: gcr.io/redskyops/result-exporter:1.1.0
+        image: gcr.io/stormforge/result-exporter:1.1.0
```

Comment:
These images don't exist @jgustie. Should we continue to use `redskyops`?

Reply:
We should probably re-build the images as `ghcr.io/thestormforge/examples-result-exporter:[latest|1.1.0]` and stop using GCR all together.

Reply:
Can you get that done @jgustie so I can update this PR to use the newer registry?